### ISSUE TITLE
Simple Framing bread support

### ISSUE THREAD

**Issue by thisisACat5** [TheIllusiveC4/CulinaryConstruct#36]
Quite self explanatory, I think Simple Farming's breads would be neat to have.

**Comment by TheIllusiveC4** [comment]
What versions do you have and which breads have you tried? As far as I can tell, it's already supported.

**Comment by thisisACat5** [comment]
Banana Bread, Sorghum Bread, Zucchini Bread, Cornbread, also Peanut Bread and Peanutbutter Bread from PeanutCraft, dont work. I think it would be nice to have them. Additionally jam would be nice to be supported.

**Comment by TheIllusiveC4** [comment]
Those breads aren't tagged as bread by Simple Farming, so that behavior is intentional. If you want that changed, you'll have to ask the Simple Farming developer. Same with PeanutCraft. They just need to add the appropriate items to the `forge/tags/items/bread` tag. You could even do it yourself with a custom datapack if you wanted.

I don't know how jam and noodles function, but they should be supported IF they're labeled as a food item. If not, then you'd have to ask the developer to change that. Alternatively, Culinary Construct provides an API that other mods can use to specify certain integration functionality like letting non-food items be used as ingredients.

Basically, all of the components to get these items supported are available, you just need to contact the developers of those other mods so they can add the appropriate integration.
